File: js/task1/S1_VAR_02.js
try {
    {
        let blockLetVar = "I'm inside the block!";
    }
    console.log(blockLetVar); 
} catch (error) {
    console.log("Friendly message: The variable is not accessible outside its block.");
}

try {
    {
        var blockVar = "I'm also inside the block!";
    }
    console.log(blockVar); 
} catch (error) {
    console.log("Friendly message: The variable is not accessible outside its block.");
}

File: python/task1/S1_VAR_02.py
try:
  def block():
      blockLetVar = "I'm inside the block!"
  block()
  print(blockLetVar)
except Exception:
  print("Friendly message: The variable is not accessible outside its block.")

try:
  blockVar = "I'm also inside the block!"
  print(blockVar)
except Exception:
  print("Friendly message: The variable is not accessible outside its block.")

print(blockVar)

try:
  print(blockLetVar)
except Exception:
  print("Friendly message: The variable is not accessible outside its block.")
